doc/release/README.multilocale

```diff
@@ -69,16 +69,17 @@ To run Chapel on multiple locales, the following steps are required:
 
 5) Set up GASNet environment variables for execution:
 
-   (a) Set up variables telling GASNet how to spawn program instances.
-       Most of our experience to date has been with two options:
+   (a) If you are using GASNet's udp conduit, set up variables telling GASNet
+       how to spawn program instances.  Most of our experience to date has been
+       with two options:
 
        (i) To run on a network of workstations to which you have
            access via ssh, configure GASNet as follows:
 
              export GASNET_SPAWNFN=S
              export GASNET_SSH_SERVERS="host1 host2 host3 ..."
-             export SSH_CMD=ssh
-             export SSH_OPTIONS=-x
+             export GASNET_SSH_CMD=ssh
```

Comment:
I like this change -- it's always mildly bothered me these two didn't have the GASNET_ prefix, but I'd never realized it wasn't necessary that it not be there!